src/Umbraco.Cms.ManagementApi/ViewModels/Move/MoveViewModel.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Runtime.Serialization;
+
+namespace Umbraco.Cms.ManagementApi.ViewModels.Move;
+
+/// <summary>
+///     A model representing a model for moving or copying
+/// </summary>
+[DataContract(Name = "content", Namespace = "")]
+public class MoveViewModel
+{
+    /// <summary>
+    ///     The Id of the node to move or copy to
+    /// </summary>
+    [DataMember(Name = "parentId", IsRequired = true)]
```

Comment:
Might be a bit redundant with your newest PR with conventions, but we should use `System.Text.Json` properties